src/main/java/game/Scene.java

```diff
@@ -0,0 +1,98 @@
+package game;
+
+import java.util.List;
+
+public class Scene implements Interactable {
+
+    private int id;
+    private String name;
+    private String description;
+    private List<String> items;
+    private Actions actions;
+
+    /*
+    User Interaction Methods
+     */
+
+    /**
+     * onCommandEntered is called when the user enters a command and determines if the command can be handled by this
+     * Scene.
+     * @param command the {@link Command} in question
+     * @param callback
+     * @return true if command has been handled, false if not.
+     */
+    @Override
+    public boolean onCommand(Command command, Callback callback) {
+
+        if (command.getAction().equals("inspect") && !command.hasReceiver()) {
+
+            StringBuilder message = new StringBuilder("inspect:\n");
+            message.append(name).append("\n\n");
+            message.append(description).append("\n\n");
+            message.append("scene commands:\n");
+            listCommands(command.getGame()).forEach(s -> message.append("\t").append(s).append("\n"));
+            message.append("player commands:\n"); // TODO: 06-03-2020 handle listing player commands somewhere else, this is messy
+            command.getGame().getPlayer()
+                    .listCommands(command.getGame()).forEach(s -> message.append("\t").append(s).append("\n"));
```

Comment:
I am not sure myself, but is there no better way to append to the message without repeating the append method?

Reply:
Using a StringBuilder is the most efficient way of making a complex string. Chaining `append()` methods is ok